Wilson.sln

```diff
@@ -97,6 +97,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.IdentityModel.Val
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.IdentityModel.SampleTests", "test\Microsoft.IdentityModel.SampleTests\Microsoft.IdentityModel.SampleTests.csproj", "{578FDF8F-6568-448A-AB93-D94269593932}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.IdentityModel.Abstractions", "src\Microsoft.IdentityModel.Abstractions\Microsoft.IdentityModel.Abstractions.csproj", "{8057C69A-3D1E-46A3-86E4-E6B26249DD25}"
```

Comment:
I noticed that you added the `strongNameBypass` reg settings. I don't think these are needed anymore. We should just strong name all our assemblies with a key that we commit to the repo. 

You can do this by adding a `Directory.Build.props` file properties in the root directory: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/Directory.Build.props#L4
and checking in this key: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/build/MSAL.snk

Reply:
IdentityModel also has 'SignAssembly' as 'true', and uses a public key (AssemblyOriginatorKeyFile). The difference seem to be in DelaySign (true in IdentityModel, false in MSAL).